### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by VortexAcherontic**
As title 😆

### CODE REVIEW

File: po/de_DE/LC_MESSAGES/de_DE.po [hunk 1]
```diff
@@ -7,140 +7,258 @@ msgid ""
...
+#. TRANSLATORS: Button Label
+#: src/main.rs:266
 msgid "Open Terminal"
 msgstr "Konsole öffnen"
 
-#: src/main.rs:207
+#. TRANSLATORS: Button Label
+#: src/main.rs:279
 msgid "Upgrade Box"
 msgstr "Box aktuallisieren"
```
Comment (ZackFairTheOne) [inline po/de_DE/LC_MESSAGES/de_DE.po:68]: "Box aktualisieren"

File: po/de_DE/LC_MESSAGES/de_DE.po [hunk 2]
```diff
@@ -7,140 +7,258 @@ msgid ""
...
+#. TRANSLATORS: Button tooltip
+#: src/main.rs:125
 msgid "About BoxBuddy"
 msgstr "Über BoxBuddy"
 
-#: src/main.rs:106
+#. TRANSLATORS: Button tooltip - Re-render application
+#: src/main.rs:132
+msgid "Reload"
+msgstr "Aktuallisieren"
```
Comment (ZackFairTheOne) [inline po/de_DE/LC_MESSAGES/de_DE.po:43]: "Aktualisieren"

File: po/de_DE/LC_MESSAGES/de_DE.po (hunk 1 above)
Comment (VortexAcherontic) [reply po/de_DE/LC_MESSAGES/de_DE.po]: Fixed thank you

File: po/de_DE/LC_MESSAGES/de_DE.po (hunk 2 above)
Comment (VortexAcherontic) [reply po/de_DE/LC_MESSAGES/de_DE.po]: Fixed as well, thank you

**Comment by Dvlv** [comment]
Thanks!

**Comment by ZackFairTheOne** [comment]
No Problem.
If you ever need a translator / newbie, shout out.

**Pull request merged**